README.md

```diff
@@ -167,7 +167,7 @@ The following document has been modified on 18th October 2018
 
 ### Social Media Coordinator
 
-[Role](https://github.com/auvzhcet/Documentation/blob/master/Social_Media_Manager.md)
+[Role](https://github.com/auvzhcet/Documentation/blob/master/docs/marketing/Social_Media_Manager.md)
```

Comment:
Again! This is the github link.
This should be [Role] (docs/marketing/Social_Media_Manager.md)

Make this change to other references also.